src/sage/rings/finite_rings/conway_polynomials.py

```diff
@@ -45,19 +48,19 @@ def conway_polynomial(p, n):
 
     EXAMPLES::
 
-        sage: conway_polynomial(2,5)
+        sage: conway_polynomial(2,5)                                                    # needs conway_polynomials
         x^5 + x^2 + 1
```

Comment:
Shouldn't the tag be file-scoped?

Reply:
This module also contains code for computing pseudo-Conway polynomials, which has to be used when the database is not available

Reply:
OK.